authorization/v2/uniformResourceIdentifier.go

```diff
@@ -52,3 +110,30 @@ func getRequestArguments(ctx *gin.Context) map[string]string {
 
 	return args
 }
+
+func unmarshalArguments(source string) (map[string]string, error) {
+	arguments := strings.Split(source, "&")
```

Comment:
Could we maybe do this inside `unmarshallURIList`?

Reply:
So, it could be moved inside `unmarshallURIList`, but the function will probably need to be renamed to something along the lines of `unmarshallDataPairs`?

Reply:
I think we could just remove `unmarshalArguments` and just use `unmarshallURIList`.

 URI lists look like this: `key1=val1&key2=val2&...` so I think it would make sense if a function called `unmarshallURIList` would unmarshall a string like the one I mentioned before.